answerking_app/utils/mixins/RetireMixin.py

```diff
@@ -20,21 +14,19 @@ def retire(self, request: Request, *args, **kwargs) -> Response:
         instance: Category | Product = self.get_object()
         if instance.retired:
             raise ProblemDetails(
-                status=status.HTTP_410_GONE,
+                status=status.HTTP_404_NOT_FOUND,
```

Comment:
why have you changed this?

Reply:
sorry I thought is the other way round. Could we change this back to 410 so it matches the current API contract otherwise we may forget about it.